charts/README.md

```diff
@@ -49,7 +49,7 @@ To disable Orbit, use:
 If a custom Orbit endpoint is required, use:
 
 ```shell
---set cloudserver-front.orbit.endpoint=https://dev.zenko.io
+--set cloudserver-front.orbit.endpoint=https://custom.orbit.endpoint
```

Comment:
If a custom endpoint is provided then a custom push endpoint must be used as well so the example should reflect that. Chances are that the push endpoint will not have access to the same data as the custom endpoint.